apps/www/app/routes/components.progress-donut.tsx

```diff
@@ -372,6 +372,21 @@ export default function Page() {
 								</p>
 							</PropDescriptionCell>
 						</PropRow>
+						<PropRow>
```

Comment:
nit, can we alphasort this to be at the top of the props table? 🙏 

<img width="1059" height="631" alt="Image" src="https://github.com/user-attachments/assets/989de812-3051-43e0-b3ab-df10209eb2c8" />